src/clib/lib/enkf/enkf_state.cpp

```diff
@@ -190,7 +191,16 @@ static fw_load_status enkf_state_load_gen_data_node(
     const enkf_config_node_type *config_node, int start, int stop) {
     fw_load_status status = LOAD_SUCCESSFUL;
     for (int report_step = start; report_step <= stop; report_step++) {
-        if (!enkf_config_node_internalize(config_node, report_step))
+
+        bool enkfConfigNodeInternalize =
```

Comment:
Style:

```suggestion
        bool enkf_config_node_internalize =
```

or maybe

```suggestion
        bool should_internalize =
```

Reply:
No objections here. :+1: